influxdb/1.2/alpine/init-influxdb.sh

```diff
@@ -30,10 +30,12 @@ if ( [ ! -z "$INIT_USERS" ] || [ ! -z "$INFLUXDB_DB" ] || [ "$(ls -A /docker-ent
 		INIT_QUERY="SHOW DATABASES"
 	fi
 
-	influxd "$@" &
+	INFLUXDB_INIT_PORT="8686"
```

Comment:
On further thought, I think this is a bad idea. If someone writes an sh script and calls `influx`, it will fail. So using the default port is probably best here and, if someone has an issue with checking the port to see if it's working, they can deal with it if it comes up.